packages/opensrp-plans/src/helpers/dataLoaders.ts

```diff
@@ -26,6 +27,11 @@ import { COULD_NOT_LOAD_ASSIGNMENTS } from '../lang';
 
 const sessionSelector = makeAPIStateSelector();
 
+export interface TaskCount {
+  total_records: string;
+  tasks: number[];
```

Comment:
I just want to confirm that the proper interface for tasks here is `number`, it seems semantically wrong, if we do not know the type of the taks field, I would suggest using `@onaio/utils.Dictionary`